src/site/includes/header.html

```diff
@@ -90,15 +90,23 @@
   <a class="show-on-focus" href="#content">Skip to Content</a>
 
   {% if !noHeader %}
-  {% include "src/site/includes/top-nav.html" %}
+    {% include "src/site/includes/top-nav.html" %}
   {% endif %}
 
   {% include "src/site/components/fullwidth_banner_alerts.drupal.liquid" %}
 
   {% if homepage_banner and homepage_banner.visible %}
-    {% include "src/site/includes/homepage-banner.liquid" with banner = homepage_banner %}
+    <div
+      data-widget-type="homepage-banner"
+      data-banner-visible="{{ homepage_banner.visible }}"
+      data-banner-title="{{ homepage_banner.title | escape }}"
+      data-banner-content="{{ homepage_banner.content | escape }}"
+      data-banner-type="{{ homepage_banner.type }}">
+    </div>
   {% endif %}
 
+  <div data-widget-type="maintenance-banner"></div>
```

Comment:
This is where the magic happens 🧙

Reply:
I'm wondering if we should consider how we could share the banner component between the homepage banner and the maintenance banner, possibly by passing props via the node dataset. What do you think?